src/aiida_quantumespresso/calculations/pw.py

```diff
@@ -168,6 +168,7 @@ def validate_inputs_base(value, _):
             if parameters.get('ELECTRONS', {}).get('startingpot', 'file') != 'file':
                 return f'`startingpot` should be set to `file` for a `{calculation_type}` calculation.'
             if parameters.get('CONTROL', {}).get('restart_mode', 'from_scratch') != 'from_scratch':
+                print(parameters)
```

Comment:
```suggestion
```

Reply:
Removed. 
I also make a mistake by pushing directly to `main`. Hopefully, (and please pretend) you didn't notice @sphuber :crying_cat_face:

Reply:
![I know nothing](https://i.pinimg.com/originals/31/1f/e4/311fe468b42dace6de2e60adefc53918.jpg)